Packs/Trellix_ePO/README.md

```diff
@@ -0,0 +1,54 @@
+3. Click the **New Server** button and select "Syslog Server" as the **Server type**, name it and click next.
+4. On the Server name field please enter the IP address or fully qualified domain name (FQDN) of your broker-vm.
+5. Specify the port through which the ePO will send logs to the broker-vm, the default port is 6514 and it only support syslog event forwarding via TLS protocol.
+6. Check the **Enable event forwarding** box.
+7. Press the **Test connection** button, if the test was successful - click the save button.
+
+* Please note that the test connection stage will only work after you finish the configuration on the broker-vm side and open the selected port on your Firewall.
+ 
+For more information check this article <https://kcm.trellix.com/corporate/index?page=content&id=KB87927>.
+ 
+### Cortex XSIAM side
+To create or configure the Broker VM, use the information described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Configure-the-Broker-VM).
+ 
+### Broker VM
+Follow the below steps to configure the Broker VM to receive Trellix ePO logs.
+ 
+1. Navigate to **Settings** → **Configuration** → **Data Broker** → **Broker VMs**.
+2. Go to the **APPS** column under the **Brokers** tab and add the **Syslog** app for the relevant broker instance. If the **Syslog** app already exists, hover over it and click **Configure**.
+3. Click **Add New**.
+4. When configuring the Syslog Collector, set the following parameters:
+    -----------------------------------------------------------------------------------------------------------------------------------------------------------
+    | Parameter: :            | Value :                                                                                                                       |
+    |-------------------------|-------------------------------------------------------------------------------------------------------------------------------|                 
+    | `Protocol`              | Select **Secure TCP**.                                                                                                        |
+    | `Port`                  | Enter the syslog service port that Cortex XSIAM Broker VM should listen on for receiving forwarded events from Trellix ePO.   |
+    | `Vendor`                | Enter trellix.                                                                                                                |
+    | `Product`               | Enter epo.                                                                                                                    |
+    | `Server Certificate`    | Select the .crt file you created, advise the attached Trellix documentation for help with using openssl.                      |
+    | `Private Key`           | Same as `Server Certificate`, select the .key file you created.                                                               |
```

Comment:
Is the same as Server Certificate? I am not sure what that means. Should the description be just: Select the .key file you created.